### DISCUSSION TITLE
SRT loads only in Russian

### DISCUSSION THREAD

**Discussion by vdgle** [URUWorks/TeroSubtitler#189] [General]
I can't find the way to open the srt in my language.
The SRT file is in Greek but it loads in Russian. 
What can I do?
![Screenshot 2023-10-19 020138](https://github.com/URUWorks/TeroSubtitler/assets/125897185/ca66b936-503a-4932-9aa0-37f463f8fce2)
![Screenshot 2023-10-19 020647](https://github.com/URUWorks/TeroSubtitler/assets/125897185/341f609f-c4c0-4262-ad3e-10d288e72276)

**Comment by URUWorks** [comment]
It is possible in a few steps,

-   Have "use Tero file dialog" enabled in Settings.
-   "File/Load subtitle..." and select the correct Encoding, "Windows-1253" if I'm not mistaken.

Can you attach the srt file?

**Comment by vdgle** [comment]
Thank you very mutch. With this way working
